dime-coding-standards/checklists/tex/hfcs_checklist.tex

```diff
@@ -0,0 +1,282 @@
+                
+            \end{itemize}
+            
+            \dimeCheckBox{\textbf{Distribution of key variables:}} 
+            The high-frequency checks contain/export:
+            
+            \begin{itemize}
+            
+                \item[]\dimeCheckBox{Table with summary stats for all key variables, highlighting missing values and outliers} 
+                
+                \item[]\dimeCheckBox{Spreadsheet listing the errors in the data that need to be shared with the field team for corrections}
+                
+                \item[]\dimeCheckBox{Bar graphs/k-density curves for \textbf{continuous variables}}
+                
+                \item[]\dimeCheckBox{Graph for number of missing and extended missing responses (like \texttt{Do not know}, \texttt{Refuse to answer}) by question type }
+                
+                \item[]\dimeCheckBox{Graph for number of outliers by question type}
+                
+            \end{itemize}
+
+            \dimeCheckBox{\textbf{Inconsistency in responses -}} 
+            The high-frequency checks contain/export:
+            \begin{itemize}
+            
+                \item[]\dimeCheckBox{Spreadsheets listing inconsistent responses with questions and their values that needs to be shared with the field team for corrections. For example, inconsistencies across variables for the same respondent, or for the same variables across different time periods.}
+                
+            \end{itemize}
+
+            \dimeCheckBox{\textbf{Form version used -}} 
+            The high-frequency checks contain/export:
+            \begin{itemize}
+            
+                \item[]\dimeCheckBox{Table listing number of responses by form-version and date}
+                
+            \end{itemize}
+
+            \end{itemize}
+
+\subsection*{Enumerator productivity}
+Tracking \textbf{enumerator productivity} involves checks for the following:
+
+            \begin{itemize}
+
+            \item[]\dimeCheckBox{\textbf{Response rates -}} 
+            The high-frequency checks contain/export:
+            
+            \begin{itemize}
+            
+                \item[] \dimeCheckBox{Table on number/share of surveys attempted, completed, and declined - for each enumerator}
+                
+                \item[] \dimeCheckBox{Table on number of attempts made to contact each interviewee - for each enumerator}
+                
+                \item[] \dimeCheckBox{Graph for total surveys attempted, completed, and declined - for each enumerator}
+                
+            \end{itemize}
+
+
+            \item[]\dimeCheckBox{\textbf{Duration -}} 
+            The high-frequency checks contain/export:
+            
+            \begin{itemize}
+            
+                \item[] \dimeCheckBox{Bar graph of average duration over each enumerator}
+                
+                \item[] \dimeCheckBox{Box plots of duration for each module over each enumerator}
```

Comment:
this is too detailed. just need to review whether they are checking duration by enumerator but i don't think we need to mandate how they are doing that.